contrib/seaweedfs/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - pschoen-itsc
```

Comment:
I think you have to comment yourself out until you become a member of the kubeflow organizations. Otherwise the tests will fail with invalid owner file.